contrib/benitlux/src/server/benitlux_db.nit

```diff
@@ -173,7 +173,7 @@ DISTINCT day FROM daily WHERE beer=(SELECT ROWID FROM beers WHERE name="{{{beer.
 	fun subscribers: Array[String]
 	do
 		var subs = new Array[String]
-		for row in select("email FROM subscribers") do subs.add row[0].to_s
+		for row in select("email FROM subscribers").as(not null) do subs.add row[0].to_s
```

Comment:
Looking at the git blame for `contrib/benitlux/src/benitlux_db.nit`:

* `Alexis Laferrière 2014-07-18 02:31:36` code with warnings introduced
* `Alexis Laferrière 2015-08-06 15:30:58` code opened
* `Alexis Laferrière 2016-04-08 11:08:41` code opened again

I'm glad the warnings act as reminders. Do you even `nitc -W`?